docs/attribution/0001-analytics.rst

```diff
@@ -283,6 +283,52 @@ To use this method you will need to include ``datalayer-begincheckout-init.es6.j
     </a>
 
 
+Product Download
+~~~~~~~~~~~~~~~~
+
+.. Important::
+
+    Only Firefox and Pocket are currently supported. VPN support has not been added.
+
+We are using a the custom event `product_download` to track product downloads (Firefox and VPN). We are not using
```

Comment:
I'm a little confused by these opening lines, it says only Firefox and Pocket are supported, but the next line says Firefox and VPN? Which is right?

Also, by Pocket, do you mean in pocket mode? (if so, should that be covered in the pocket mode section?)

Reply:
Clear now?

Reply:
It doesn't yet seem super clear to me where we are and aren't using it from the docs, but I'm guessing this is also documentation for work that's still in progress, so let's move forward and update as we go? :)